src/main/java/org/cloudfoundry/community/servicebroker/s3/config/ServiceConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package org.cloudfoundry.community.servicebroker.s3.config;
+
+import org.cloudfoundry.community.servicebroker.model.Plan;
+import org.cloudfoundry.community.servicebroker.model.ServiceDefinition;
+import org.cloudfoundry.community.servicebroker.s3.plan.basic.BasicPlan;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by jcarter on 22/01/16.
+ */
+@Configuration
+public class ServiceConfiguration {
+
+    @Value("${SERVICE_ID:s3}")
+    private String serviceId;
+
+    @Value("${SERVICE_NAME:amazon-s3}")
+    private String serviceName;
+
+    @Value("${PLAN_ID:s3-basic-plan}")
+    private String basicPlanId;
+
+    @Value("${PLAN_NAME:basic}")
+    private String basicPlanName;
+
+    @Bean
+    public ServiceDefinition serviceDefinition() throws IOException {
+        return new ServiceDefinition(serviceId, serviceName,
+                "Amazon S3 is storage for the Internet.", true, getPlans(), getTags(), getServiceDefinitionMetadata(),
+                Arrays.asList("syslog_drain"), null);
+    }
+
+    private List<String> getTags() {
+        return Arrays.asList("s3", "object-storage");
```

Comment:
Can you add `"blob"` to this please?

Then it might "just work" with https://github.com/cloudfoundry-community/vcap-ruby/blob/master/auto-reconfiguration/lib/cf-autoconfig/blob/aws_s3.rb#L13